paparazzi/src/main/java/app/cash/paparazzi/InstantAnimationsRule.kt

```diff
@@ -45,6 +46,7 @@ public class InstantAnimationsRule : TestRule {
     }
     set(value) {
       setDurationScale.invoke(null, value)
+      PaparazziMotionDurationScale.animationScaleFactor = value
```

Comment:
If I'm understanding this correctly, a non-Compose project applying this rule would result in a ClassNotFoundException, right?

Reply:
possibly, should add tests to ensure this.